microbial_thermodynamics/cell_growth_ODE.py

```diff
@@ -0,0 +1,143 @@
+"""Summary:
+This calculates rates of change of population growth, ribosome fraction and
+internal energy production.
+
+Desctiption:
+I am uncertain if the R in calculate_lam and the r in dr are the same thing (ribosome
```

Comment:
They are the same thing as the growth rate (lambda) depends on the ribosome fraction (R), the dr function is used to track changes in the ribosome fraction. It might make sense to switching to using either r or R throughout